Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -168,6 +179,70 @@ public final class Chat {
         try await channelUpdater.removeMembers(currentUserId: currentUserId, cid: cid, userIds: Set(members), message: systemMessage)
     }
     
+    /// Loads an array of members for the specified query and updates ``ChatState.members``.
+    ///
+    /// - Note: Channel member sorting keys are set when creating the ``Chat`` instance.
+    /// It is also possible to create separate ``MemberList`` objects if needed with different filtering options. See ``ChatClient.makeMemberList(with:)``.
+    ///
+    /// - Parameter pagination: The pagination configuration which includes a limit and an offset or a cursor.
+    ///
+    /// - Throws: An error while communicating with the Stream API.
+    /// - Returns: An array of paginated channel members for the query.
+    public func loadMembers(with pagination: Pagination) async throws -> [ChatChannelMember] {
+        try await memberList.loadMembers(with: pagination)
+    }
```

Comment:
I tried out what we discussed which is Chat having a way to have channel members as part of the state but still having a way to create separate MemberLists. My proposal is that the `Chat` one only supports setting sorting options (like messages) but if one creates a separate MemberList, then all the filtering and sorting is available.